docs/contribution_handling.md

```diff
@@ -0,0 +1,55 @@
+# WinUI Repo Contribution Handling
+
+The WinUI repo is intended as a place for the WinUI team to gather community feedback, discuss issues with the community, and provide insight into bug fixes that the team is working on before updates are released. Here, we'll outline how we handle feature requests and bugs that the community opens.
+
+## Issues
+
+Feature requests and bugs are tracked as GitHub issues.
+
+For reporting security issues please see the [Security Policy](SECURITY.md).
+
+For all other bugs and general issues please [file a new issue](https://github.com/Microsoft/microsoft-ui-xaml/issues/new/choose) using the Bug Report template.
+
+Please file questions and discussions as discussions, not issues.
+
+## Feature Proposals
+
+Feature proposals for WinUI 3 are categorized based on the likely timeframe in which the feature team can consider them.
+
+1. 	Short-term, less than a year
+2.	Long-term, not likely in the next year
+
+All feature proposals are automatically categorized as long-term and unlikely to be considered, although the WinUI team might change the priority based on customer and business needs. If the WinUI team does close a feature proposal, we will indicate the reason why and how it fits into our plans. Closing a proposal does not mean that the team will never consider it. We encourage the community to add comments or interact with a proposal at any time, even a closed one, to signal its importance to the team.
+
+### Feature Planning
+
+Our long term vision for WinUI is to be the best UX/UI native framework for Windows PCs that provides premium visuals, motion, accessibility, usability, and support for all input modalities.
```

Comment:
Thank you for adding what is somewhat of a vision statement. It's nice to know what drives your overall priorities, albeit I'm a bit sad to see developer experience isn't part of it. It's all imho a bit moot if we give up half-way to getting there.

Reply:
I would be very much appreciated if the roadmap for developers can be defined, too.  At present, the information about native Windows development is confusing, to put it mildly.  The various options are lacking their properly defined places, and WinUI 3 is not making is better, especially when the current state of the SDK and the development tools feel more like a beta than a production-ready product.